wgsl/index.bs

```diff
@@ -8411,41 +8595,64 @@ Unless explicitly permitted below, an attribute [=shader-creation error|must not
 
 </table>
 
+## Shader Stage Attributes ## {#shader-stage-attr}
+
 The <dfn noexport>shader stage attributes</dfn> below
 designate a function as an [=entry point=] for a particular [=shader stage=].
 These attributes [=shader-creation error|must=] only be applied to [=function declarations=],
 and at most one may be present on a given function.
 They take no parameters.
 
-<table class='data'>
-  <caption>Shader Stage Attributes</caption>
-  <thead>
-    <tr><th>Attribute<th>Description
-  </thead>
+### `vertex` ### {#vertex-attr}
+
+<pre class=include>
+path: syntax/vertex_attr.syntax.bs.include
+</pre>
 
-  <tr><td><dfn noexport dfn-for="attribute">`vertex`</dfn><br>
+<table class='data builtin'>
+  <caption><dfn noexport dfn-for="attribute">`vertex`</dfn> Attribute</caption>
+  <tr>
+    <td style="width:10%">Description
     <td>Declares the function to be an [=entry point=] for the [=vertex shader stage=]
     of a [=GPURenderPipeline|render pipeline=].
 
-  <tr><td><dfn noexport dfn-for="attribute">`fragment`</dfn><br>
+</table>
+
+### `fragment` ### {#fragment-attr}
```

Comment:
Here is a case (shader stages) I would consider forgoing consistency. When I look at the rendered spec, the table seems unnecessary altogether. Maybe it would be better to drop the table and just keep the description text.

Reply:
I think we had a similar discussion at an editors' call before, I also think that such tabularization bloats a bit, especially harming mobile reading experience. I changed a bit in the latest commit, PTAL! Thank you.